CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Use rate-law function `GCARR_ac` for rxns that have Arrhenius `B` parameters that are zero
 - Now use correct index `WEAEROSOL(I,J,L,2+NDUST)` in routine `Settle_Strat_Aer` of `GeosCore/ucx_mod.F90`
 - Now get density of BCPI species from the species database in `ucx_mod.F90`
+- Fix issues that prevented single-species carbon simulations from running
+- Update `HEMCO_Config.rc.carbon` and `ExtData.rc.carbon` templates for consistency
```

Comment:
It looks like this PR has some other things, like removing activate_tagged_species. Should that be included in the removed list in the changelog, or did that originally come in recent dev?

Reply:
I updated CHANGELOG.md in https://github.com/geoschem/geos-chem/pull/1992/commits/e837aba3ec9d112b4ab4cc501707e621e43db042.